CONTRIBUTING.md

```diff
@@ -22,11 +22,27 @@ Some general rules to follow:
 - Create a branch for each update that you're working on. These branches are often called "feature" or "topic" branches. Any changes that you push to your feature branch will automatically be shown in the pull request.
 - Keep your pull requests as small as possible. Large pull requests are hard to review. Try to break up your changes into self-contained and incremental pull requests.
 - The first line of commit messages should be a short (&lt;80 character) summary, followed by an empty line and then any details that you want to share about the commit.
+- Each pull request should be associated with an issue.
 - Please try to follow [common commit message conventions](https://chris.beams.io/posts/git-commit/).
 
+### Voting
+Once a pull request or issue have been submitted, maintainers can comment or vote on to express their opinion following the [Apache voting system](https://www.apache.org/foundation/voting.html). Quick summary:
+
+- +1 something you agree with
```

Comment:
Can we equate a PR approval with a +1 vote and a PR "request changes" with a -1 vote?

Reply:
Good idea!

Reply:
@jfuerth How's this?

Reply:
LGTM